helm/defectdojo/values.yaml

```diff
@@ -250,8 +250,14 @@ django:
     # could be emptyDir (not for production) or pvc
     type: emptyDir
     # in case if pvc specified, should point to the already existing pvc
-    persistentVolumeClaim: media
-
+    persistentVolumeClaim:
+      # set to true if django.mediaPersistentVolume.type is set to pvc
+      create: false
+      name:
+      size: 10Gi
+      accessModes:
+      - ReadWriteOnce
```

Comment:
For more than one replica ReadWriteOnce would not work.

Reply:
What do you suggest it should be for more than one replica? 
Wouldn't it be ok to just have it commented in front as "#change to (e.g ReadWriteMany) when using more than one replica".  Please feel free to put it in better way so that I can update everything accordingly

Reply:
Do you see disadvantages when using ReadWriteMany as default?

Reply:
No disadvantage that I can think of at the moment. I will go ahead to have this modified and push the commit.